### ISSUE TITLE
Accessibility: student feedback

### ISSUE THREAD

**Issue by amyjko** [amyjko/user-interface-software-and-technology#76]
Three ways in which the chapter can be improved:

+ Make a stronger case for accessibility. So far, the case for accessibility in the chapter is presented from a humanitarian point of view, but other reasons should be brought up as well, such as business reasonings. The main reason a stronger case for accessibility is needed is that even though the seemingly impressive statistics, it is just not a very convincing number. 15% means more than one out of ten have some problems, but from real life experience, it doesn't really feel like that. 

+ It is worth mentioning that not all accessibility problems are equal. For example, I have a form of red color blind, and a pretty serious case as well. However, I have been using everything like normal my entire life. But people with impaired vision, for example, will not be able to use computers easily. 

+ Add in a good example of a universal interface. Otherwise, it is really difficult to visualize what an universal design is supposed to look like. If such an example doesn't exist, it also worths mentioning why universal design is impossible.

**Comment by andyjko** [comment]
> The chapter mentioned 15% of human population experiences disability of some sorts. However this is a very heterogeneous population in terms of disability distribution, I would be curious to see a breakdown of the types of disability with respect to % of population. The World Bank link currently does not include this information. 

> A quick Google search returns many different players (Links to an external site.) playing games competitively using prototyped physical accessibility technologies given their physical limitations. The chapter mentioned making use of full senses, it would be interesting to see any academic research that discussed the modification of hardware to suit accessibility needs. 

> The sample list of access technologies provided was a good way to help understand the concept using tools people are already familiar with. It would be a good idea to provide a few more examples (screen reader and captions are great), or at least to substitute braille, since it's not very commonly seen.
